spartanstay/spartanstay/src/main/java/com/spartanstay/spartanstay/service/ListingsServiceImpl.java

```diff
@@ -11,14 +11,15 @@
 public class ListingsServiceImpl implements ListingsService{
 
     @Override
-    public String getListings(String destId, String checkIn, String checkOut, String sortOrder, String adults) {
+    public String getListings(String destId, String checkIn, String checkOut, String sortOrder, String adults, String starRatings) {
         HttpRequest request = HttpRequest.newBuilder()
                 .uri(URI.create("https://hotels4.p.rapidapi.com/properties/list?destinationId="+ destId +
                         "&pageNumber=1&pageSize=25" +
                         "&checkIn="+checkIn+
                         "&checkOut="+checkOut+
                         "&adults1=" + adults +
                         "&sortOrder=" + sortOrder +
+                        "&starRatings=" + starRatings +
```

Comment:
And then here we would want some conditional logic to check if this parameter was received before making this https request.